README.md

```diff
@@ -11,7 +11,8 @@ This repo only contains the source SVG files. Check out [maki website](https://m
 
 - Source icons are in the SVG file format.
 - Icons are available in two sizes: 11px x 11px and 15px x 15px.
-- Icons are each a single unified SVG path, with no transforms, groups or strokes.
+- Icons should consist only of paths and groups. Paths should only have a `d` property.
+- Each Icon's svg tag should have an `id` property that corresponds to it's filename without the extension.
```

Comment:
Grammar/style recommendations:

* `Icon's` => `icon's`
* `it's filename` => `its filename`